config/features/config.go

```diff
@@ -122,6 +122,7 @@ func configureTestnet(ctx *cli.Context) error {
 		if err := params.SetActive(params.RopstenConfig().Copy()); err != nil {
 			return err
 		}
+		enableDevModeFlags(ctx)
```

Comment:
I'm not sure about enabling all the dev flags. How about just enabling the ones that we need?

Reply:
Dev is only two flags afaik

Reply:
The issue is someone could add any flags to dev later that we may not want them on a production testnet. I would prefer this to be flag specific